src/javascript/DesignSystem/DatePicker/DatePicker.jsx

```diff
@@ -39,37 +39,83 @@ const DatePickerCmp = ({
     classes,
     onSelectDateTime,
     selectedDateTime,
+    disabledDays: incomingDisableDays,
     ...props
 }) => {
     const [selectedDays, setSelectedDays] = useState([]);
-    const [selectedHour, setSelectedHour] = useState('0:00');
+    const [selectedHour, setSelectedHour] = useState('00:00');
+    const isDateTime = variant === 'datetime';
 
     const locale = locales[lang] || {};
 
     const selectedDate = selectedDateTime ? [selectedDateTime] : selectedDays;
-    const isDateTime = variant === 'datetime';
+    let disabledHours = {};
+    const disabledDays = [];
+    const extractDateAndHours = (entry, boundIncludeOffset) => {
+        return {
+            date: !isDateTime && entry.include ? entry.date : undefined,
+            hours: isDateTime && dayjs(entry.date).format('YYYYMMDD') === dayjs(selectedDate).format('YYYYMMDD') ? dayjs(entry.date).add(entry.include ? boundIncludeOffset : 0, 'minute').format('HH:mm') : undefined
```

Comment:
It not necessary to add 0 here in this code: dayjs(entry.date).add(entry.include ? boundIncludeOffset : 0, 'minute')
could be changed to:
entry.include ? dayjs(entry.date).add(boundIncludeOffset, 'minute').format('HH:mm') : dayjs(entry.date).format('HH:mm')

Also dayjs(entry.date) is already called before, so it's possible to store the result of: dayjs(entry.date) in a variable and reuse it instead of calling it multiple times